packages/wrangler/src/tail/printing.ts

```diff
@@ -24,13 +24,13 @@ export function prettyPrintLogs(data: WebSocket.RawData): void {
 
   if (eventMessage.logs.length > 0) {
     eventMessage.logs.forEach(({ level, message }) => {
-      console.log(`  (${level}) ${message}`);
+      console.log(`  (${level})`, message);
```

Comment:
this fix prevents `[Object object]` being printed for complex objects

Reply:
This was the one I thought would most benefit from a test.